api/client/trust.go

```diff
@@ -425,6 +425,10 @@ func (cli *DockerCli) trustedPush(repoInfo *registry.RepositoryInfo, tag string,
 	}
 
 	err = repo.Publish()
+	if err == nil {
+		fmt.Fprintf(cli.out, "Finished signing %q with tag %s\n", repoInfo.FullName(), tag)
```

Comment:
As @HuKeping says, It looks like this code only runs if a tag was specified at push time. I don't really understand why it works this way - an untrusted push without an explicit tag will push all tags for that repo, and I would expect trusted push to behave the same way.